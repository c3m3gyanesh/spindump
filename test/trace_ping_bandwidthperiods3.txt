Testing the --bandwidth-period option with PING traffic. In this third test, we set the bandwidth period to 10s (10 000 000 us).
