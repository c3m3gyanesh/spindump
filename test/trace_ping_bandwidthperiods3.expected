HOSTS 10.0.1.36 <-> 212.16.100.1 0 sessions new static packets 0 0 bytes 0 0
HOSTS 10.0.1.36 <-> 212.16.100.1 1 sessions at 1571746859995979 measurement static right 7720 packets 1 0 bytes 1028 0 bandwidth 103 0
HOSTS 10.0.1.36 <-> 212.16.100.1 1 sessions at 1571746861000447 measurement static right 7180 packets 2 1 bytes 2056 1028 bandwidth 206 103
HOSTS 10.0.1.36 <-> 212.16.100.1 1 sessions at 1571746862006744 measurement static right 8825 packets 3 2 bytes 3084 2056 bandwidth 308 206
HOSTS 10.0.1.36 <-> 212.16.100.1 1 sessions at 1571746863012223 measurement static right 13015 packets 4 3 bytes 4112 3084 bandwidth 411 308
HOSTS 10.0.1.36 <-> 212.16.100.1 1 sessions at 1571746864012797 measurement static right 8429 packets 5 4 bytes 5140 4112 bandwidth 514 411
HOSTS 10.0.1.36 <-> 212.16.100.1 1 sessions at 1571746865083568 measurement static right 78815 packets 6 5 bytes 6168 5140 bandwidth 617 514
HOSTS 10.0.1.36 <-> 212.16.100.1 1 sessions at 1571746866073640 measurement static right 65993 packets 7 6 bytes 7196 6168 bandwidth 720 617
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746866567563 measurement static right 7009 packets 8 7 bytes 7524 7196 bandwidth 752 720
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746866685711 measurement static right 21227 packets 9 8 bytes 7852 7524 bandwidth 785 752
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746866781081 measurement static right 13549 packets 10 9 bytes 8180 7852 bandwidth 818 785
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746866880129 measurement static right 8126 packets 11 10 bytes 8508 8180 bandwidth 851 818
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746866988835 measurement static right 12213 packets 12 11 bytes 8836 8508 bandwidth 884 851
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867016944 measurement static right 6874 packets 13 12 bytes 9864 8836 bandwidth 986 884
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867093603 measurement static right 12450 packets 14 13 bytes 10192 9864 bandwidth 1019 986
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867192190 measurement static right 6472 packets 15 14 bytes 10520 10192 bandwidth 1052 1019
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867302207 measurement static right 12820 packets 16 15 bytes 10848 10520 bandwidth 1085 1052
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867403250 measurement static right 9037 packets 17 16 bytes 11176 10848 bandwidth 1118 1085
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867505429 measurement static right 6401 packets 18 17 bytes 11504 11176 bandwidth 1150 1118
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867611040 measurement static right 7463 packets 19 18 bytes 11832 11504 bandwidth 1183 1150
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867712058 measurement static right 7095 packets 20 19 bytes 12160 11832 bandwidth 1216 1183
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867813890 measurement static right 7910 packets 21 20 bytes 12488 12160 bandwidth 1249 1216
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746867912959 measurement static right 6742 packets 22 21 bytes 12816 12488 bandwidth 1282 1249
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868015184 measurement static right 8586 packets 24 22 bytes 14172 12816 bandwidth 1417 1282
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868022058 measurement static right 7129 packets 24 23 bytes 14172 13144 bandwidth 1417 1314
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868113706 measurement static right 6910 packets 25 24 bytes 14500 14172 bandwidth 1450 1417
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868220121 measurement static right 8994 packets 26 25 bytes 14828 14500 bandwidth 1483 1450
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868326403 measurement static right 10848 packets 27 26 bytes 15156 14828 bandwidth 1516 1483
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868425920 measurement static right 6624 packets 28 27 bytes 15484 15156 bandwidth 1548 1516
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868527014 measurement static right 7034 packets 29 28 bytes 15812 15484 bandwidth 1581 1548
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868631222 measurement static right 7519 packets 30 29 bytes 16140 15812 bandwidth 1614 1581
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868734718 measurement static right 7262 packets 31 30 bytes 16468 16140 bandwidth 1647 1614
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868840186 measurement static right 8235 packets 32 31 bytes 16796 16468 bandwidth 1680 1647
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746868941992 measurement static right 7841 packets 33 32 bytes 17124 16796 bandwidth 1712 1680
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869027413 measurement static right 7236 packets 34 33 bytes 18152 17124 bandwidth 1815 1712
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869043651 measurement static right 8469 packets 35 34 bytes 18480 18152 bandwidth 1848 1815
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869144709 measurement static right 8122 packets 36 35 bytes 18808 18480 bandwidth 1881 1848
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869247361 measurement static right 6583 packets 37 36 bytes 19136 18808 bandwidth 1914 1881
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869353790 measurement static right 8636 packets 38 37 bytes 19464 19136 bandwidth 1946 1914
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869459182 measurement static right 9512 packets 39 38 bytes 19792 19464 bandwidth 1979 1946
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869579158 measurement static right 26126 packets 40 39 bytes 20120 19792 bandwidth 2012 1979
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869678332 measurement static right 22820 packets 41 40 bytes 20448 20120 bandwidth 2045 2012
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869765061 measurement static right 7131 packets 42 41 bytes 20776 20448 bandwidth 2078 2045
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869870354 measurement static right 7684 packets 43 42 bytes 21104 20776 bandwidth 2110 2078
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746869972700 measurement static right 7656 packets 44 43 bytes 21432 21104 bandwidth 2143 2110
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870030888 measurement static right 6692 packets 45 44 bytes 22460 21432 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870072571 measurement static right 6423 packets 46 45 bytes 22788 22460 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870181837 measurement static right 12355 packets 47 46 bytes 23116 22788 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870318586 measurement static right 44406 packets 48 47 bytes 23444 23116 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870383306 measurement static right 7150 packets 49 48 bytes 23772 23444 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870485843 measurement static right 7067 packets 50 49 bytes 24100 23772 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870590045 measurement static right 6665 packets 51 50 bytes 24428 24100 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870694291 measurement static right 7135 packets 52 51 bytes 24756 24428 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870800182 measurement static right 10012 packets 53 52 bytes 25084 24756 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746870917136 measurement static right 24162 packets 54 53 bytes 25412 25084 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746871002984 measurement static right 6061 packets 55 54 bytes 25740 25412 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746871038427 measurement static right 9670 packets 56 55 bytes 26768 25740 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746871130720 measurement static right 28651 packets 57 56 bytes 27096 26768 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746871225081 measurement static right 19285 packets 58 57 bytes 27424 27096 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746871330125 measurement static right 21110 packets 59 58 bytes 27752 27424 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746871438565 measurement static right 26718 packets 60 59 bytes 28080 27752 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746871520221 measurement static right 7190 packets 61 60 bytes 28408 28080 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746871624971 measurement static right 9566 packets 62 61 bytes 28736 28408 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746872040820 measurement static right 8050 packets 63 62 bytes 29764 28736 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746873045443 measurement static right 8051 packets 64 63 bytes 30792 29764 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746874049578 measurement static right 9599 packets 65 64 bytes 31820 30792 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746875086518 measurement static right 43962 packets 66 65 bytes 32848 31820 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746876054329 measurement static right 7900 packets 67 66 bytes 33876 32848 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746877055031 measurement static right 7179 packets 68 67 bytes 34904 33876 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746878061232 measurement static right 8181 packets 69 68 bytes 35932 34904 bandwidth 2143 2143
HOSTS 10.0.1.36 <-> 212.16.100.1 2 sessions at 1571746879066743 measurement static right 8928 packets 70 69 bytes 36960 35932 bandwidth 2143 2143
